ir-engine/ir-engine
# feat(studio): create a project history after update the static resource [IR-5106]

## Summary

- Add to the project history when the user adds a new asset thumbnail

## Subtasks Checklist

## Breaking Changes

## References
closes #_insert number here_
https://github.com/ir-engine/ir-engine/blob/87adccb126a55f1eb31296bad8f81849ad9bcc14/packages/client-core/src/common/services/FileThumbnailJobState.tsx#L171

## QA Steps

1. Open Studio
2. Open a scene
3. Navigate to the Files panel
4. Upload a file from the local machine
5. Observe thumbnail creation for an uploaded file
6. Navigate to Dashboard
7. Click History on the corresponding project
8. Check logs including file uploading action
9. Check logs including thumbnail creation action for uploaded file

## Jira
[IR-5106](https://tsu.atlassian.net/browse/IR-5106)


[IR-5106]: https://tsu.atlassian.net/browse/IR-5106?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
